frontend/src/App.vue

```diff
@@ -33,7 +33,7 @@ const apiUrl = inject(apiUrlKey);
 const route = useRoute();
 const routeName = typeof route.name === 'string' ? route.name : '';
 const router = useRouter();
-const lang = defaultLocale();
+// const lang = defaultLocale();
```

Comment:
This is just to get rid of the linting warning. Do we want to keep that code?

Reply:
We can remove, or add `_` in front of lang.